vo-dml/mango.vo-dml.xml

```diff
@@ -1146,6 +1161,20 @@ As long as the vocabulary is not set, the possible values of this attribute are
           <maxOccurs>1</maxOccurs>
         </multiplicity>
       </attribute>
+      <attribute>
```

Comment:
I'm guessing that this metadata hasn't been needed in practice for the use cases, but added after re-reading the confidenceLevel discussion from Meas RFC1. ??  If this is associated with the confidenceLevel, they should be encapsulated in an object/datatype.